### PR TITLE
Add management command to add a refendeum election

### PR DESCRIPTION
**Pull request by michaeljcollinsuk**
- Added to allow us to add Croydon referendum initially, with
the possibility of being able to use again in the future as required

### CODE REVIEW

File: every_election/apps/elections/management/commands/add_referendum.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            required=True,
+        )
+        parser.add_argument(
+            "--org-type",
+            action="store",
+            dest="org-type",
+            default="local-authority",
+            help="The council area the referendum is taking place",
+            required=True,
+        )
```
Comment (michaeljcollinsuk) [inline every_election/apps/elections/management/commands/add_referendum.py:59]: If I have gone overboard with making these arguments I can hardcode them

**Comment by symroe** [comment]
This looks decent. At the moment it only deals with referendums that cover the _whole_ organisation, not a division. It's worth at least adding that in the help text, but if possible allowing an optional `--division_id` that would allow adding a division.

When we do point in polygon lookups we try to use the division first, so this should be all that's needed.

File: every_election/apps/elections/management/commands/add_referendum.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            election=ref_election,
+        )
+
+        org = Organisation.objects.get_by_date(
+            date=election_date,
+            official_identifier=options["official_identifier"],
+            organisation_type=options["org_type"],
+        )
+        ref_election.organisation = org
+        ref_election.organisation_geography = org.geographies.latest()
```
Comment (symroe) [inline every_election/apps/elections/management/commands/add_referendum.py:99]: I think you need to set the division geography on the table too?

Comment (michaeljcollinsuk) [reply every_election/apps/elections/management/commands/add_referendum.py:99]: Sorry, unfamiliarity with the data model! Added in latest commit - if that looks right I can rebase before merging

**Review by symroe — APPROVED** [review]

**Pull request merged**